backend/src/ml_space_lambda/utils/image_uris.py

```diff
@@ -31,6 +31,46 @@
 
 logger = logging.getLogger(__name__)
 
+### START CUSTOM MLSPACE ADDITIONS FOR DETECTING BUILT-IN ALGORITHMS
+# Most images are named the same as their frameworks except for the below exceptions
+IMAGE_TO_FRAMEWORK_NAME_CONVERSIONS = {"sagemaker-xgboost": "xgboost"}
```

Comment:
It doesn't look like any part of this custom addition actually relies on internal methods in this file and only leverages `retrieve` which we could call externally. Given that this file is basically a copy from the SageMaker SDK I think it'd be better to keep this in it's own utility file (or just outside of this file). When OCI container support is available in region we'll be able to bundle the SageMaker SDK directly.